Microsoft Windows [VersiÃ³n 10.0.19044.1526]
(c) Microsoft Corporation. Todos los derechos reservados.

C:\Users\helru>cd
C:\Users\helru

C:\Users\helru>C:\Users\helru\kata2
"C:\Users\helru\kata2" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

C:\Users\helru>cd C:\Users\helru\kata2

C:\Users\helru\kata2>python3 -m venv env

C:\Users\helru\kata2>source env/bin/activate
"source" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

C:\Users\helru\kata2>env\Scripts\activate

(env) C:\Users\helru\kata2>pip freeze

(env) C:\Users\helru\kata2>pip install python-dateutil
Collecting python-dateutil
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting six>=1.5
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: six, python-dateutil
Successfully installed python-dateutil-2.8.2 six-1.16.0
WARNING: You are using pip version 21.2.4; however, version 22.0.3 is available.
You should consider upgrading via the 'C:\Users\helru\kata2\env\Scripts\python.exe -m pip install --upgrade pip' command.

(env) C:\Users\helru\kata2>pip freeze
python-dateutil==2.8.2
six==1.16.0

(env) C:\Users\helru\kata2>deactivate
C:\Users\helru\kata2>
